series/series.go

```diff
@@ -231,6 +313,26 @@ func Bools(values interface{}) Series {
 	return New(values, Bool, "")
 }
 
+// StringsList is a constructor for an StringList Series
+func StringsList(values interface{}) Series {
```

Comment:
Why is it called `StringsList` instead of `StringList`? Seems inconsistent with the type-name. It' also applicable for `IntsList`,`FloatsList`, and `BoolsList` .

Reply:
Because StringList already used as type name:

```
// Supported Series Types
const (
	String     Type = "string"
	Int        Type = "int"
	Float      Type = "float"
	Bool       Type = "bool"
	StringList Type = "string_list"
	IntList    Type = "int_list"
	FloatList  Type = "float_list"
	BoolList   Type = "bool_list"
)```

For constructor, I followed previous naming: `Strings()` for constructing `string` series type, hence `StringsList()` for `string_list`